openlibrary/core/schema.sql

```diff
@@ -107,3 +107,22 @@ CREATE TABLE wikidata (
     data json,
     updated timestamp without time zone default (current_timestamp at time zone 'utc')
 )
+
+CREATE TABLE bestbooks (
```

Comment:
`book_id` will need to become `work_id` in our code so that it works with the resolve redirect logic

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/core/db.py#L25-L46

The type should also be `integer` similar to the e.g. star ratings table:

https://github.com/internetarchive/openlibrary/blob/4e761dd9e792d87b7fc8f6a56a0bf13cb11533cd/openlibrary/core/schema.sql#L4